### PR TITLE
Migrate "how to contribute" page from old wiki

### PR DESCRIPTION
**Pull request by f18m**
Problem:
* all libzmq documentation pages are pointing to the "contribution policy page" http://zeromq.wikidot.com/docs:contributing which lands on the "old" zeromq website; this can be misleading for newcomers, given that the old zeromq website is full of outdated references and information. 

Solution:
* Just migrate the content from old "as is" from the old wiki to the new website

### CODE REVIEW

**Comment by f18m** [comment]
@sappo , @somdoron , @bluca ,

Let me know if such approach (moving the page) is feasible for you.
If it is, then I would ask you to add my wikidot account "f18m" to the list of wikidot site administrators/maintainers; that will allow me to click "Edit" in the wikidot page and grab the source of the page, making the translation work "old->new website" easier.
This is what I get clicking on the "Edit" button of http://zeromq.wikidot.com/docs:contributing page:

image

Thanks

**Comment by bluca** [comment]
The page to add new members doesn't work anymore on wikidot, at least for me - searching for a user to invite does nothing

**Comment by f18m** [comment]
> The page to add new members doesn't work anymore on wikidot, at least for me - searching for a user to invite does nothing

Ok I see. In reality I already have my user "f18m" registered... I just need some kind of permissions to edit the "protected pages" I guess.

Then just let me know if adding a new page like I'm proposing is fine, I can take the time to put the right Markdown formatting in place (currently the formatting in this PR is very rough). 

Last consideration: I think the frontpage of the wiki (which used to be the homepage of zeromq project) is misleading... there is a lot of outdated stuff that I think should be hidden. The risk is to confuse newcomers because if they hit a link going to the wiki, and they start to navigate from there, they will find a lot of duplicated content from the new website (the download page, references to the API, etc etc).
Long story short: we need somebody (and I might volunteer to do that) to cleanup all the pages from the old wiki that have been moved to the new website, I think.

**Comment by f18m** [comment]
hi @sappo ,
thanks for the merge!
However to be honest the MR was not 100% ready... I will open a follow up to fix the formatting...
